File: Game/firstGame.js
const prompt=require("prompt-sync")();
let name=prompt("Enter Ur Name:");
console.log(`Welcome ${name}`);
let res=1;
let num=Math.floor(Math.random()*(100-1))+1;
while(true){
    let input=prompt("Enter number b/w 1 to 100-> ");
    if (input>num){
        console.log("Too High");
    } else if (input<num){
        console.log("Too Low");
    }
    else{
        console.log("Congratulations "+name+", You guessed it! in "+ res +" Times");
        break;
    }
    res++;
}

File: Java/Game/firstGame.java
package Java.Game;
import java.util.Scanner;

public class firstGame {
    static void main(){
        Scanner NameInput=new Scanner(System.in);
        System.out.println("Enter Full Name:");
        String playerName=NameInput.nextLine();
        if (playerName.length()<1){
            System.out.println("Invalid Input");
            main();
        }
        else {
            System.out.println("Welcome "+playerName+" to the game!");
        }
        NameInput.close();
        int randomNum12 = (int)(Math.random()*(100 - 1))+1;
        int count=1;
        while(true){
            Scanner userInput=new Scanner(System.in);
            System.out.println("Guess a number between 1 and 100:");
            int guess=userInput.nextInt();
            if (guess>randomNum12){
                System.out.println("Too high!");
            }
        else if (guess<randomNum12){
            System.out.println("Too low!");
        }
        else {
            System.out.println("Congratulations! You guessed the correct number in "+count+" times.");
            break;
        }
        count++;
    }
}
}